PR for SRVLOGIC-530: Add the section to upgrade Serverless Logic Operator from 1.35.0 to 1.36.0 (#93332)
openshift/openshift-docs: modules/serverless-logic-upgrade-1.36-restoring-workflows-with-preview-profile.adoc

// Module included in the following assemblies:
// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36

:_mod-docs-content-type: PROCEDURE
[id="serverless-logic-upgrade-1.36-restoring-workflows-with-preview-profile_{context}"]
= Restoring workflows with the preview profile

> Workflows with the `preview` profile require an additional configuration step before being redeployed.

Reviewer: ```suggestion
After the upgrade, you must redeploy workflows that use the `preview` profile and any associated Kubernetes resources.
```

.Procedure

> 

Reviewer: @kaldesai  @gmunozfe

I think guys we don't need this because we are treating the update from 1.35.0 to 1.36.0, meaning that, If you are using persistence in 1.35.0 you must already have been set that  kie.flyway.enabled=true in your workflows.

That  kie.flyway.enabled property, was introduced for updates from 1.34.0  to 1.35.0

> 

Reviewer: @domhanak  In the migration guide, I believe that no. I think it can confuse people that are just migrating their installation in 1.35.0 to 1.35.0

From the migration guide point of view, I believe it should be like this.

I have workflows already created in 1.35.0?

ok, if you used persistence, it means that you have already created and configured them properly. And, from the migration perspective you don¡'t need do add any configuration.

I want to start a WF from scratch?

where, in 1.35.0? I need to read the workflows configuration section, etc,  and know how to configure the persistence.

where, in 1.36.0? Same as :point_up: , but if I am already in 1.36.0, a migration o a brand new installation was already performed.

> . Ensure all required resources are recreated, including `SonataFlow` CRDs, `ConfigMaps` with `application.properties`, or any other related custom configurations before you redeploy the workflow.

Reviewer: ```suggestion
. Ensure all required Kubernetes resources, including the `ConfigMap` with the `application.properties`, are restored before redeploying the workflow.
```

. Redeploy the workflow by running the following command: 
+
[source,terminal]
----
$ oc apply -f <my-workflow.yaml> -n <target_namespace>

> ----

Reviewer 2: Same suggestions as earlier
